supply/spec/client_spec.rb

```diff
@@ -40,6 +40,8 @@
         expect(subject.class.method_defined?(:get_edit_listing)).to eq(true)
         expect(subject.class.method_defined?(:list_edit_apks)).to eq(true)
         expect(subject.class.method_defined?(:list_edit_bundles)).to eq(true)
+        expect(subject.class.method_defined?(:list_generatedapks)).to eq(true)
+        expect(subject.class.method_defined?(:download_generatedapk)).to eq(true)
```

Comment:
Are these correct?

Reply:
Yes. That spec tests methods we expect to be defined **on the `AndroidPublisher::AndroidPublisherService` internal client** (not to be confused with the `Supply::Client`).

Those two particular ones that I added there are documented here in the `AndroidPublisherService` ruby doc:
 - [`AndroidPublisher::AndroidPublisherService#list_generatedapks`](https://googleapis.dev/ruby/google-apis-androidpublisher_v3/v0.33.0/Google/Apis/AndroidpublisherV3/AndroidPublisherService.html#list_generatedapks-instance_method)
 - [`AndroidPublisher::AndroidPublisherService#download_generatedapk`](https://googleapis.dev/ruby/google-apis-androidpublisher_v3/v0.33.0/Google/Apis/AndroidpublisherV3/AndroidPublisherService.html#download_generatedapk-instance_method)

And those are the ones that I call in the `Supply::Client` corresponding wrapper methods [here](https://github.com/fastlane/fastlane/pull/21315/files#diff-f291ca0a82f6a4ac1725fec2caa16f022ec8a2852841fa13f3016dea73584484R316) and [here](https://github.com/fastlane/fastlane/pull/21315/files#diff-f291ca0a82f6a4ac1725fec2caa16f022ec8a2852841fa13f3016dea73584484R332), respectively.

So all that is correct (and consistent with how the rest of the spec and that file is written) 🙂

Reply:
Ah I see. Thanks! I'm much less familiar with the Android code.

Reply:
No worries! Took me some time to familiarize myself with the AndroidPublisher API and Supply code and specs as well when I wrote that PR 😅 🙂